src/test/java/org/fundacionjala/pivotal/steps/api/APIProjectSteps.java

```diff
@@ -12,17 +15,26 @@
  * Class with steps for test feature.
  */
 public class APIProjectSteps {
+    private final APIResponseName date;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "date" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)